### PR TITLE
upgrade to backstage 1.26.5

### PR DESCRIPTION
**Pull request by nabuskey**
Backend migration to the new backend system is mostly done. The ArgoCD plugin doesn't have support for the new backstage backend yet so I had to use the legacy method.

### CODE REVIEW

**Review by nimakaviani — APPROVED** [review]
LGTM.

thanks for this.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# CNOE [Backstage](https://backstage.io)
+# CNOE Backstge
```
Comment (omrishiv) [inline README.md:1]: @nabuskey `Backstage`

File: packages/app/src/apis.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ import {OAuth2} from "@backstage/core-app-api";
 export const keycloakOIDCAuthApiRef: ApiRef<
   OpenIdConnectApi & ProfileInfoApi & BackstageIdentityApi & SessionApi
 > = createApiRef({
-  id: 'auth.keycloak-oidc-provider',
+  id: 'auth.keyloak-oidc',
```
Comment (omrishiv) [inline packages/app/src/apis.ts:16]: `keycloak`

Comment (nabuskey) [reply packages/app/src/apis.ts:16]: Good catch. I gotta update

**Pull request merged**